### ISSUE TITLE
Mask R-CNN

### ISSUE THREAD

**Issue by joyhuang9473** [number9473/nn-algorithm#63]
# Mask R-CNN #

- Author: Kaiming He, Georgia Gkioxari, Piotr Dollár, Ross Girshick
- Origin: https://arxiv.org/abs/1703.06870v1
- Related:
  - https://github.com/facebookresearch/Detectron
  - [https://github.com/matterport/Mask_RCNN](https://github.com/matterport/Mask_RCNN)
  - [https://github.com/CharlesShang/FastMaskRCNN](https://github.com/CharlesShang/FastMaskRCNN)
  - https://github.com/jasjeetIM/Mask-RCNN
  - https://github.com/DeNA/Chainer_Mask_R-CNN
  - https://github.com/TuSimple/mx-maskrcnn
  - [Facebook 何凯明发大招：Mask R-CNN 狙击目标实例分割 ](https://mp.weixin.qq.com/s?__biz=MzA4NzE1NzYyMw==&mid=2247488392&idx=2&sn=7c8e41aef37c370d6155607283d776ef&chksm=903ced90a74b6486a7bc76619a6fe10b8b92e50a0e7c7ac1825c0b54b515cc5b4b9e8b433ceb&mpshare=1&scene=1&srcid=0328OXU5TNCFPSkYNysGujxM#rd)

**Comment by joyhuang9473** [comment]
Mask R-CNN for object detection and instance segmentation on Keras and TensorFlow

https://github.com/matterport/Mask_RCNN

**Comment by lzx1413** [comment]
https://github.com/TuSimple/mx-maskrcnn

**Comment by joyhuang9473** [comment]
https://github.com/facebookresearch/Detectron

#144 (Faster R-CNN)
#156 (RetinaNet)
